security/experimental_authenticators.rst

```diff
@@ -135,6 +135,42 @@ unauthenticated access (e.g. the login page):
             ],
         ]);
 
+Granting Anonymous Users Access in a Custom Voter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.2
+
+    The ``NullToken`` class was introduced in Symfony 5.2.
+
+If you're using a :doc:`custom voter </security/voters>`, you can allow
+anonymous users access by checking for a special new
```

Comment:
I'd remove "new" because this is "new" only in this Symfony version, but not in next versions.